### ISSUE TITLE
errors if comma-spacing isn't respected in template strings if placeholders are used in it

### ISSUE THREAD

**Issue by bloodyowl** [eslint/eslint#2393]
```javascript
const foo = `hello,`

console.log(foo)
```

doesn't error, when

```javascript
const foo = `hello${ 1 + 1 },`

console.log(foo)
```

does error

```
1:28  error  A space is required after ','  comma-spacing
```

**Comment by nzakas** [comment]
So what are you reporting? Are you saying the second example should not error?

What version of ESLint are you using? Are you using the default parser or something like babel-eslint?

**Comment by bloodyowl** [comment]
the second example is wrong in the sense the lint shouldn't bother checking *inside* strings.

my bad, I'm using the babel-eslint@2.0.2 parser. reporting that to the right repository.
